byron/ledger/formal-spec/delegation.tex

```diff
@@ -329,131 +333,62 @@ \subsection{Delegation sequences}
   \begin{equation}
     \inference
     {
+      {\begin{array}{l}
+         delegEnv
```

Comment:
I'm not 100% sure, but I think you should wrap `delegEnv` and such with a `\var{}`. This affects rendering.

Reply:
Yup, without the `\var` macro the characters are rendered slightly separated, as if they were individual variables.

Reply:
fixed